src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -42,28 +72,59 @@ const EditTestCase = () => {
 
   return (
     <>
-      <Button
-        tw="m-2"
-        variant="cyan"
-        type="submit"
-        data-testid="qdm-test-case-run-button"
-        onClick={calculateQdmTestCases}
-      >
-        QDM Run Execution
-      </Button>
-      <Toast
-        toastKey="edit-action-toast"
-        aria-live="polite"
-        toastType={toastType}
-        testId={toastType === "danger" ? "error-toast" : "success-toast"}
-        closeButtonProps={{
-          "data-testid": "close-toast-button",
-        }}
-        open={toastOpen}
-        message={toastMessage}
-        onClose={onToastClose}
-        autoHideDuration={10000}
+      <EditTestCaseBreadCrumbs
+        testCaseTitle={currentTestCase?.title}
+        testCaseId={currentTestCase?.id}
+        measureId={measureId}
       />
+      <form id="edit-test-case-qdm">
+        <div className="allotment-wrapper">
+          <Allotment
+            proportionalLayout={true}
+            separator={true}
+            vertical={false}
```

Comment:
Looks like proportionalLayout and separator defaulted to true, do we still need them? Ignore me if they are not, I am speaking solely based on Allotment docs.

Reply:
also, Do you think the default split should happen 1/3 -2/3? That's what I remember from mock ups

Reply:
missed this. updated accordingly